packages/api-client/src/views/Request/RequestSection/RequestAuth/RequestAuthTab.vue

```diff
@@ -165,17 +165,26 @@ const dataTableInputProps = {
   <template
     v-for="{ scheme } in security"
     :key="scheme?.uid">
-    <!-- Header/Description -->
-    <DataTableRow
-      v-if="
-        security.length > 1 || (scheme?.description && security.length <= 1)
-      ">
+    <!-- Header -->
+    <DataTableRow v-if="security.length > 1">
       <DataTableCell
-        :aria-label="generateLabel(scheme!) || scheme?.description"
+        :aria-label="generateLabel(scheme!)"
+        class="text-c-2 group/auth flex items-center leading-[22px] whitespace-nowrap outline-none hover:whitespace-normal">
+        <p
+          class="bg-b-1 text-c-2 outline-b-3 top-0 z-1 h-full w-full overflow-hidden px-3 py-1.25 text-ellipsis group-hover/auth:absolute group-hover/auth:h-auto group-hover/auth:border-b *:first:line-clamp-1 *:first:text-ellipsis group-hover/auth:*:first:line-clamp-none">
+          {{ generateLabel(scheme!) }}
```

Comment:
I wonder why we need the !, maybe we should add a v-if="scheme" somewhere higher, or make the function deal with an `undefined` input